### PR TITLE
Replace Digits popup with embed widget

### PR DESCRIPTION
**Pull request by alexsinger**
New embed widget:

cannonballapp embed widget

### CODE REVIEW

**Comment by mylyuic** [comment]
@alexsinger , thanks for the change.

 i think it's still going to be useful to keep the old pop-up mode as well since it would be convenient for us to verify the pop-up behavior with this sample app. It's also a useful pop-up mode reference implementation for developers as well.

**Comment by alexsinger** [comment]
@mylyuic hmm ok. How do you envision that example? Having two different pages: one with pop-up and one with embed?

**Comment by mylyuic** [comment]
In the past, we started the pop-up flow by clicking on a button, we can retain the button and its functionality and add the new embed widget for the embedding flow so that we have both.

**Comment by andypiper** [comment]
I think having two example pages may be the least confusing way of presenting the options.

**Comment by mylyuic** [comment]
@alexsinger , any update on this change?

File: public/javascripts/cannonball.js [hunk 1]
```diff
@@ -82,12 +80,10 @@
   }
 
   // Set the Digits button label (and make sure it is not disabled).
-  function setDigitsButton(text) {
-    $('.digits-button').text(text).removeAttr('disabled');
+  function setDigitsNumber(text) {
```
Comment (missing-semicolon) [inline public/javascripts/cannonball.js:83]: Good rename.

**Pull request closed without merging**